code/modules/antagonists/voidwalker/voidwalker.dm

```diff
@@ -0,0 +1,50 @@
+/// Space antagonist that harasses people near space and cursed them if they get the chance
+/datum/antagonist/voidwalker
+	name = "\improper Voidwalker"
+	antagpanel_category = ANTAG_GROUP_ABOMINATIONS
+	job_rank = ROLE_VOIDWALKER
+	show_in_antagpanel = FALSE
+	show_name_in_check_antagonists = TRUE
+	show_to_ghosts = TRUE
+	ui_name = "AntagInfoVoidwalker"
+	suicide_cry = "FOR THE VOID!!"
+	preview_outfit = /datum/outfit/voidwalker
+
+/datum/antagonist/voidwalker/greet()
+	. = ..()
+	owner.announce_objectives()
+
+/datum/antagonist/voidwalker/on_gain()
```

Comment:
Maybe we should also set the species here?
I am thinking:"What would an admin expect the behaviour to be when applying this in the antag panel?" and that probably includes the species.